src/pathfora.js

```diff
@@ -2426,10 +2583,9 @@
      * @public
      * @description Initialize Pathfora widgets from a container
      * @param {object|array}   widgets
-     * @param {string}         lyticsId
      * @param {object}         config
      */
-    this.initializeWidgets = function (widgets, lyticsId, config) {
+    this.initializeWidgets = function (widgets, config) {
```

Comment:
this is totally fine / much better but we should try and make it as backwards compatible as possible. should be able to just check if that second argument is actually a config and not a string, if its a string we should assume its an id and just throw it out and look to the 3rd argument for the config.

definitely should update all of our internal stuff to correctly handle this but difficult to ensure everyone that could be using this will also update.